cds-service/Package.md

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Shall we have this as a README.md

Reply:
Fixed: https://github.com/wso2/solutions-healthcare-demos/pull/3/commits/c41a84abbe219e92e2f234f8546f47f49c3bb3e7